microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -1,17 +1,33 @@
 ---
-description: Progressive Web Apps (Chromium) run natively on Windows 10.  Here is everything you need to know as a web developer.
-title: Progressive Web Apps on Windows
+description: Progressive Web Apps run natively on Windows 10.  Here is everything you need to know as a web developer.
+title: Overview of Progressive Web Apps (PWAs)
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 05/06/2021
+ms.date: 09/10/2021
 ms.topic: article
 ms.prod: microsoft-edge
 ms.technology: pwa
-keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft Store
+keywords: progressive web apps, PWA, Edge, JavaScript, UWP, Microsoft Store
 ---
-# Progressive Web Apps on Windows overview
+# Overview of Progressive Web Apps (PWAs)
 
-[Progressive Web Apps][MDNApps] \(PWAs\) provide access to open web technologies for cross-platform interoperability and provide your users with a native, app-like experience customized for their devices. PWAs are websites that are [progressively enhanced][AListApartUnderstandingProgressiveEnhancement] to function like native apps on supporting platforms. The qualities of a PWA combine the best of the web and native apps.
+**[Progressive Web Apps][MDNApps]** (PWAs) provide access to open web technologies for cross-platform interoperability and provide your users with a native, app-like experience customized for their devices.
+
+PWAs are websites that are **[progressively enhanced][AListApartUnderstandingProgressiveEnhancement]** to function like native apps on supporting platforms, and like regular web sites on other browsers.
+
+The qualities of a PWA combine **the best of the web and native apps**. PWAs run in browsers, like websites, but have access to native app features like the ability to work offline, be installed as a native app, support push notifications and periodic updates, access hardware features, and more.
+
+**PWAs are first-class citizens on Windows**. When installed, they behave like other apps. They can be added to the Start Menu, pinned to the Taskbar, handle files, run on user login, and more.
+
+PWAs can also be **submitted to the Microsoft Store** where millions of Windows users can discover and easily install them alongside other Windows apps.
+
+:::image type="complex" source="./media/windows-apps-and-store.png" alt-text="PWAs on Windows, in Microsoft Edge, and the Microsoft Store" lightbox="./media/windows-apps-and-store.png":::
+    PWAs on Windows, in Microsoft Edge, and the Microsoft Store
+:::image-end:::
+
+## Characteristics of a PWA
+
+PWAs have the same reach as websites: they can be indexed by search engines, can be linked to, and work on all devices from the same codebase. As such they have a much lower development cost than native apps that require a specific codebase  for Android, iOS, and various desktop operating systems.
```

Comment:
I’m hearing some pushback on the term "native" from an inclusivity perspective. Alternative could be "proprietary" or "compiled."

Reply:
Thanks! I wasn't aware. I made changes to all instances of "native" in this page, and used a different phrasing depending on the context.